cmd/thanos/query.go

```diff
@@ -161,6 +161,8 @@ func registerQuery(app *extkingpin.App) {
 	enableMetricMetadataPartialResponse := cmd.Flag("metric-metadata.partial-response", "Enable partial response for metric metadata endpoint. --no-metric-metadata.partial-response for disabling.").
 		Hidden().Default("true").Bool()
 
+	activeQueryDir := cmd.Flag("query.active-query-path", "Directory to log currently active queries in the queries.active file.").Default(".").String()
```

Comment:
Does this mean that this will always be on? :thinking: I think this feature comes with some performance implications. Maybe we can disable it by default by having an empty string as the default value, for example?